cirq/circuits/circuit_test.py

```diff
@@ -918,6 +922,33 @@ def test_insert_into_range():
 y: ───────────────@───────────
 """)
 
+    c.insert_into_range([cirq.Y(y), cirq.Y(y), cirq.Y(y), cirq.CX(y, x)], 1, 4)
+    cirq.testing.assert_has_diagram(
+        c, """
+x: ───X───X───Z───@───X───X───X───
+                  │       │
+y: ───────Y───Y───@───Y───@───────
+""")
+
+    c.insert_into_range([cirq.H(y), cirq.H(y)], 6, 7)
+    cirq.testing.assert_has_diagram(
+        c, """
+x: ───X───X───Z───@───X───X───X───────
+                  │       │
+y: ───────Y───Y───@───Y───@───H───H───
+""")
+
+    c.insert_into_range([cirq.T(y)], 0, 1)
+    cirq.testing.assert_has_diagram(
+        c, """
+x: ───X───X───Z───@───X───X───X───────
+                  │       │
+y: ───T───Y───Y───@───Y───@───H───H───
+""")
+
+    with pytest.raises(IndexError):
+        c.insert_into_range([cirq.CZ(x, y)], 10, 10)
+
```

Comment:
What are these new insertion tests about?

Reply:
Our test coverage tools are flagging sections of the `circuit.py` code that I moved in this PR. I assume this code predates our enforcement of test coverage.